deployments/compose/docker-compose.yml

```diff
@@ -210,3 +210,21 @@ services:
       - ./dbt/profiles.yml:/root/.dbt/profiles.yml
     entrypoint: dbt
 
+  notebook:
+    build: notebook
+    ports:
+      - 8888:8888
+    environment:
+      - SPARK_MODE=worker
+      - SPARK_MASTER_URL=spark://spark:7077
+      - SPARK_WORKER_MEMORY=1G
+      - SPARK_WORKER_CORES=1
+      - SPARK_RPC_AUTHENTICATION_ENABLED=no
+      - SPARK_RPC_ENCRYPTION_ENABLED=no
+      - SPARK_LOCAL_STORAGE_ENCRYPTION_ENABLED=no
+      - SPARK_SSL_ENABLED=no
```

Comment:
MIght be time to move it into a configuration file or an env file

Reply:
True - I like it when it is very explicit and can update in one place. But I think it will be a good followup PR